osu.Game/Beatmaps/Formats/LegacyBeatmapDecoder.cs

```diff
@@ -67,36 +67,34 @@ protected override void ParseStreamInto(LineBufferedReader stream, Beatmap beatm
 
         protected override void ParseLine(Beatmap beatmap, Section section, string line)
         {
-            var strippedLine = StripComments(line);
-
             switch (section)
             {
                 case Section.General:
-                    handleGeneral(strippedLine);
+                    handleGeneral(line);
                     return;
 
                 case Section.Editor:
-                    handleEditor(strippedLine);
+                    handleEditor(line);
                     return;
 
                 case Section.Metadata:
                     handleMetadata(line);
```

Comment:
Perhaps the reason why it was doing it locally was because of this?

Reply:
Yeah.. on checking the usages of the `Editor` section on stable there doesn't look to be valid reasoning behind this either way.